common/services/prismic/link-resolver.ts

```diff
@@ -52,18 +55,20 @@ function linkResolver(doc: Props | DataProps): string {
   }
 
   if (type === 'pages') {
-    if ('siteSection' in doc) {
-      return `${doc.siteSection}/${uid}`;
-    } else if ('tags' in doc) {
-      // Needed for Prismic previews
-      const docSiteSection = doc.tags.find(t => isSiteSection(t));
+    let siteSection: SiteSection | undefined;
 
-      const isLandingPage = docSiteSection === uid;
-      if (isLandingPage) return `/${uid}`;
+    if ('siteSection' in doc) {
+      siteSection = doc.siteSection;
+    }
 
-      return `${docSiteSection ? '/' + docSiteSection : ''}/${uid}`;
+    // Prismic previews come through here.
+    if ('tags' in doc) {
```

Comment:
does this mean that if there is a sitesection tag, it would take precedence over `doc.siteSection` – and if so is that ok?

Reply:
I guess so yes, and if so it should still be the correct `siteSection`, so I'd say it's ok!